PluralKit.Bot/Commands/Misc.cs

```diff
@@ -214,9 +228,29 @@ public async Task PermCheckGuild(Context ctx)
                     eb.Color(DiscordUtils.Red);
                 }
             }
+            string footer = "";
+
+            foreach (var (missingPermissionField, channels) in permissionsMissing)
```

Comment:
please use LINQ :p

Reply:
> please use LINQ instead, it's a lot cleaner

Maybe respond in #internal-stuff, but definitely, I just wasn't sure how, I would really love a quick example if you have time of how to do that better, I haven't worked with LINQ much yet

Reply:
I haven't tried this, but it should work:
```cs
if (permissionsMissing.Any((missingPermissionField, _) => missingPermissionField == (ulong)PermissionSet.UseExternalEmojis)
```